### PR TITLE
Add 11.19 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

New week. New release

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [ ] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [ ] I have read the **README** document about contributing to this repo.
- [ ] I have tested my changes locally and there are no issues.
- [ ] All commits are signed.

### CODE REVIEW

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+slug: mondoo-11.19-is-out/
+title: Mondoo 11.19 is out!
+description: Announcing the 11.19 release of Mondoo with a new Shodan security policy, security policy updates, and more!
+authors: [tim]
+image: /img/releases/2024-08-27-mondoo-11.19-is-out/shodan.png
+tags: [release, mondoo]
+---
+
+## 🥳 Mondoo 11.19 is out! This release includes a new Shodan security policy, security policy updates, and more!
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:10]:
```suggestion
## 🥳 Mondoo 11.19 is out! This release includes a new Shodan security policy, updates to existing security policies, and more!
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)
+
+---
+
+## 🎉 NEW FEATURES
+
+### New Mondoo Shodan Security policy
+
+Secure critical publically facing domains and IP addresses with the new Mondoo Shodan Security policy. This policy includes checks to identify open ports and vulnerabilities as reported by the Shodan network scanning service.
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:20]:
```suggestion
Secure critical, public-facing domains and IP addresses with the new Mondoo Shodan Security policy. It includes checks to identify open ports and vulnerabilities as reported by the Shodan network scanning service.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+![Shodan policy](/img/releases/2024-08-27-mondoo-11.19-is-out/shodan.png)
+
+## 🧹 IMPROVEMENTS
+
+### Risk summaries for vendor advisories
+
+Better understand the criticality of vendor advisories with new score summaries. Advisories now include an overall risk score, taking into account blast radius and Mondoo identified risk factors.
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:28]:
```suggestion
Better understand the criticality of vendor advisories with new score summaries. Advisories now include an overall risk score, taking into account blast radius and Mondoo-identified risk factors.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Mondoo now includes vulnerability detection on Windows 10/11 Enterprise multi-session editions.
+
+### Compliance data in exports
+
+Data exports now include compliance control references, so you can feed compliance evidence collected by Mondoo into external data analytics systems.
+
+### Expanded security policies
+
+Secure your ever growing fleet of assets, with new and expanded policies:
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:42]:
```suggestion
Secure your ever growing fleet of assets with new and expanded policies:
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+- Mondoo HTTP Security policy now includes checks to ensure Server, X-AspNetMvc-Version, X-AspNet-Version, X-Powered-By, and Public-Key-Pins HTTP headers are not set
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:44]:
```suggestion
- Mondoo HTTP Security policy now includes checks to ensure Server, X-AspNetMvc-Version, X-AspNet-Version, X-Powered-By, and Public-Key-Pins HTTP headers are not set.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- CIS Azure Foundations benchmark policy includes 8 new checks as well as filter and query improvements
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:45]:
```suggestion
- CIS Azure Foundations benchmark policy includes 8 new checks as well as filter and query improvements.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+- Mondoo Endpoint Detection and Response (EDR) policy now includes support for Cortex XDR
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:46]:
```suggestion
- Mondoo Endpoint Detection and Response (EDR) policy now includes support for Cortex XDR.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+- AWS Operational Best Practices includes S3 bucket permission checks
+
+### New console audit events
+
+Keep tabs on changes made to your Mondoo organization and spaces with improved audit logging. New events included in the Mondoo Console audit log include:
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:51]:
```suggestion
Keep tabs on changes made to your Mondoo organization and spaces with improved audit logging. New events recorded in the Mondoo Console audit log include:
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+- Integrations deleted
+- Space settings modified
+- Registration tokens created
+- API tokens created
+- Compliance reports generated
+- Custom policy or framework uploads
+
+### Expanded query packs
+
+Collect important asset inventory data with new and expanded query packs.
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:67]:
```suggestion
Collect important asset inventory data with new and expanded query packs:
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+- macOS Inventory Pack now collects system extensions information.
+- Shodan inventory pack now collects DNS NS records, subdomains, and domain tag information.
+- New Event Log Collector for NTLMv1 query pack.
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:71]:
```suggestion
- There's a new Event Log Collector for NTLMv1 query pack.
```

To make it a sentence like the others

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+- New Event Log Collector for SMBv1 query pack.
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:72]:
```suggestion
- We added an Event Log Collector for SMBv1 query pack.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+### Resource updates
+
+#### github.commit
+
+- New `authoredDate` field
+- New `committedDate` field
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Add custom registry icons in the registry for AlmaLinux, BSI, Oracle Linux, Rocky Linux, and Arista.
```
Comment (misterpantz) [inline releases/2024-08-27-mondoo-11.19-is-out.md:83]:
```suggestion
- Add custom icons in the registry for AlmaLinux, BSI, Oracle Linux, Rocky Linux, and Arista.
```

File: releases/2024-08-27-mondoo-11.19-is-out.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+Data exports now include compliance control references, so you can feed compliance evidence collected by Mondoo into external data analytics systems.
+
+### Expanded security policies
+
+Secure your ever growing fleet of assets with new and expanded policies:
+
+- Mondoo HTTP Security policy now includes checks to ensure Server, X-AspNetMvc-Version, X-AspNet-Version, X-Powered-By, and Public-Key-Pins HTTP headers are not set.
+- CIS Azure Foundations benchmark policy includes 8 new checks as well as filter and query improvements.
+- Mondoo Endpoint Detection and Response (EDR) policy now includes support for Cortex XDR.
+- AWS Operational Best Practices policy now includes S3 bucket permission checks.
```
Comment (tas50) [inline releases/2024-08-27-mondoo-11.19-is-out.md:47]:
```suggestion
- AWS Operational Best Practices policy includes improved S3 bucket permission checks.
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**